face_track/face_track.py

```diff
@@ -392,8 +407,26 @@ def track_face(self, faceid):
 
 	# ----------------------------------------------------------
 
-	def user_said_cb(self,msg):
-		self.atomo.who_spoke(msg.data)
+	# def speech_event_cb(self,msg):
+	# 	if msg.data == "speechstart":
+	# 		self.speech_start_at = datetime.datetime.now()
+	# 	if msg.data == "speechend":
+	# 		self.speech_end_at = datetime.datetime.now()
+			#self.speech_elapse_ms = (self.speech_end_at - self.speech_start_at) * 1000
+ 	def stt_cb(self,msg):
+		if msg.confidence >= 50:
+			self.atomo.who_said(msg.utterance)
+
+	# def user_said_cb(self,msg):
+	# 	if msg.confidence >= 50:
+	# 		t1 = (datetime.datetime.now() - self.speech_start_at)
+	# 		t2 = (datetime.datetime.now() - self.speech_end_at)
+	# 		self.speech_elapse_ms =  t1.total_seconds()* 1000
+	# 		self.speech_end_elapse_ms = t2.total_seconds()*1000
+	# 		self.atomo.who_spoke(msg.utterance.lower(),
```

Comment:
More commented out code.

Also, what does this have to do with face-tracking?   Perhaps a different py file should be created for this speech stuff.  Integration of speech and visual perception should not be happening in the python layers, anyway.